sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/models/TilesetId.java

```diff
@@ -49,12 +49,6 @@ public final class TilesetId extends ExpandableStringEnum<TilesetId> {
     /** Static value microsoft.weather.infrared.main for TilesetID. */
     public static final TilesetId MICROSOFT_WEATHER_INFRARED_MAIN = fromString("microsoft.weather.infrared.main");
 
-    /** Static value microsoft.dem for TilesetID. */
-    public static final TilesetId MICROSOFT_DEM = fromString("microsoft.dem");
-
-    /** Static value microsoft.dem.contours for TilesetID. */
-    public static final TilesetId MICROSOFT_DEM_CONTOURS = fromString("microsoft.dem.contours");
```

Comment:
As this was public API, we should deprecate rather than remove these two fields.

Reply:
If the previous version was beta, would it be ok to remove the fields since it is being auto-generated from the updated swagger?